examples/video/instream/videoModule/videojs/video-module-videojs.html

```diff
@@ -0,0 +1,280 @@
+                  <!--put body html and javascript here-->
+                  &lt;div class="example-video-container"&gt;
+                    &lt;video id="player" class="vjs-big-play-centered"&gt;&lt;/video&gt;
+                  &lt;/div&gt;
+                </pre>
+              </div>
+            `;
+        </script>
+    </div>
+</div>
+
+{% if jekyll.environment == "production" %}
+<script>
+    window.onetrustLoaded.then(() => {
+        OneTrust.InsertHtml(exampleHTML, 'videoExample', null, {deleteSelectorContent: true}, 'C0003');
+    });
+</script>
+{% else %}
+<script>
+    const videoExampleDiv = document.getElementById('videoExample');
+    videoExampleDiv.innerHTML = exampleHTML;
+</script>
+{% endif %}
+
+<!--video player code-->
+<script>
+    // Setup ad units
+    var adUnits = [{
+        code: 'div-gpt-ad-51545-0',
+        mediaTypes: {
+            video: {}
+        },
+        video: {
+            divId: 'player', // required to indicate which player is being used to render this ad unit.
+        },
+        bids: [{
+            bidder: 'ix',
+            params: {
+                siteId: '300',
+            }
+        }]
+    }];
+
+    var pbjs = pbjs || {};
+    pbjs.que = pbjs.que || [];
+
+    pbjs.que.push(function () {
+        pbjs.setConfig({
+            video: {
+                providers: [{
+                    divId: 'player',
+                    vendorCode: 2, // videojs vendorCode
+                    playerConfig: {
+                        params: {
+                            adPluginConfig: {
+                                numRedirects: 10,
+                                adLabel: "Using Prebid Video Module!"
+                            },
+                            vendorConfig: {
+                                controls: true,
+                                autoplay: true,
+                                preload: "auto",
+                            }
+                        }
+                    },
+                    // Configure your Ad Server Integration
+                    adServer: {
+                        vendorCode: 'gam',
+                        baseAdTagUrl: 'https://pubads.g.doubleclick.net/gampad/ads?iu=/21775744923/external/single_preroll_skippable&sz=640x480&ciu_szs=300x250%2C728x90&gdfp_req=1&output=vast&unviewed_position_start=1&env=vp&impl=s&correlator='
+                        //'https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&iu=/137679306/HB_Dev_Center_Example&impl=s&gdfp_req=1&env=vp&output=vast&unviewed_position_start=1&correlator=',
+                    },
+                },]
+            },
+            cache: {
+                url: 'https://prebid.adnxs.com/pbc/v1/cache'
+            },
+            targetingControls: {
+                allowTargetingKeys: ['BIDDER', 'AD_ID', 'PRICE_BUCKET', 'SIZE', 'DEAL', 'SOURCE', 'FORMAT', 'UUID', 'CACHE_ID', 'CACHE_HOST', 'ADOMAIN']
+            },
+            debugging: {
```

Comment:
Do we need all these debugging overrides? Can't we just use the test appnexus tags?